proposals/dictionary-expressions.md

```diff
@@ -33,13 +33,20 @@ An inclusive solution is needed for C#. It should meet the vast majority of case
 The following grammar productions are added:
 
 ```diff
++ collection_expression
+  : '[' ']'
+- | '[' collection_element ( ',' collection_element )* ']'  
++ | '[' 'comparer': expression ']'
+  | '[' ('comparer': expression , )? collection_element ( ',' collection_element )* ']'
```

Comment:
Missing some quotes:
```suggestion
  : '[' ']'
- | '[' collection_element ( ',' collection_element )* ']'  
+ | '[' 'comparer' ':' expression ']'
  | '[' ('comparer' ':' expression ',' )? collection_element ( ',' collection_element )* ']'
```

This could also be:

```g4
  : '[' ('comparer' ':' expression )? ']'
  | '[' ('comparer' ':' expression ',' )? collection_element ( ',' collection_element )* ']'
```